core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -1432,6 +1432,13 @@ class EMConfig {
     var skipFailureSQLInTestFile = false
 
 
+    @Experimental
+    @Cfg("Specify a maximum number of existing data in the database to sample when SQL handling is enabled. " +
+            "Note that 0 means all existing data would be sampled")
+    @Min(0.0)
+    var maximumExistingDataToSampleInDb = 0
+
```

Comment:
hmmm... but what is one wants no data to sample (ie 0)?
maybe better like no data is sampled if the value is negative, with -1 being the default